feat(preamble): :sparkles: expand on introduction section of syllabus (#114)
rostools/github-intro: preamble/syllabus.qmd

# Syllabus {#sec-syllabus}

Over the last few decades, the way we do research has substantially
changed as the internet and, especially, digital technologies and social
media have become more embedded in even the more traditional and
conservative parts of our society. There are many ways this is impacting
research, that has placed a greater need, expectation, and/or pressure

> on a wide range of things, often at a fundamentally. Some of these needs

Reviewer: Fundamentally *what*? :)

include, in in no particular order:

1.  Formally disseminate more types of new or existing knowledge and
    scientific output
2.  Informally sharing information, discussions, or ideas more
    frequently and more rapidly
3.  Collaborate more with others across regions and timezones
4.  Be more open and transparent about the whole research process
5.  Network and connect with others in the research community and, at
    least in health, those who participate or want to participate in
    research using a wider variety of tools and platforms

While academia and research pursue expanding the boundaries of our
collective knowledge, we also paradoxically are very slow to adapt to
and adopt changes, especially technological. But the needs listed above

> are now so great that we are forced into adapting to and using these
> tools to effectively filful those needs.

Reviewer: ```suggestion
are now so great that we must embrace these tools
tools to effectively fulfil those needs.
```
To make it a bit more motivational/less “we have to do this bc we’re forced to do it”? 
I do love “filful” though 😝

Author: Ha, it's my partial dyslexia :stuck_out_tongue:

> This workshop aims to help introduce you to one such tool and way of
> working and communicating that is very common in industry and becoming
> more common in academia: GitHub. While GitHub can filful all those needs

Reviewer: ```suggestion
This workshop introduces you to one such tool and way of
working and communicating that is very common in industry and becoming
more common in academia: GitHub. While GitHub can fulfil the needs
```
